addons/website_forum/models/forum_post.py

```diff
@@ -694,21 +692,24 @@ def convert_comment_to_answer(self, message_id, default=None):
         return new_post
 
     def unlink_comment(self, message_id):
+        comment = self.env['mail.message'].sudo().browse(message_id)
```

Comment:
"message_sudo" ? "comment_sudo" ? Prefer keeping the sudo keyword to avoid calling sub methods as sudo without noticing it.